doc/devel/northboundinteractions.md

```diff
@@ -955,6 +1050,32 @@ The Context Broker replies with all the desired data, in R2 format (200 OK):
 ]
 ```
 
+#### Differences regarding the new commands mode
+
+A new commands flow has been defined (involving also modifications at ContextBroker). As part of that design, commands
+are not sent to IOTAs using NGSIv2 notifications, but the current implementation has some differences regarding the
+desired behaviour, which are described next:
+
+* Fields others than `targetEntityId`, `targetEntityType`, `cmd` and `params` (i.e. `execTs`,
+`status`, `info`, `onDelivered`, `onOk`, `onError`, `onInfo`, `cmdExecution` and `dataExpiration`), are not actually used. By
+the moment they are stored in the commands model (commands collection) but nothing is done with them appart from storing.
+* If `targetEntityId` and `targetEntityType` are not found to find device involved in command, then `id` and `type` of
+notification are used instead of them. This should not be done (as the `id` and `type` in the notification are the ones of
+a transient entity, not the entity associated to the device) but this behaviour is kept by the moment to provide a smooth
+transition to the new approach (so, at the present moment, as we don't have transient entities representing command executions,
+the command notification is sent based on a subscription done on the entity associated to the device).
```

Comment:
Maybe it's better to remove this "failsafe" behaviour so users of the new functionality doesn't get any "bad habits".

It is a matter of doing this (easy) in the sub:

```
"ngsi": {
  "targetEntityType: {
     "value": "${type}",
  },
  "targetEntityId: {
     "value": "${id}",
  }
}
```

Reply:
```suggestion
```

I understand this is no longer valid after commit https://github.com/telefonicaid/iotagent-node-lib/pull/1560/commits/c07c3c478d397a53884a444a880852bbef9fa564